src/server/server_family.cc

```diff
@@ -2385,7 +2385,7 @@ void ServerFamily::ReplConf(CmdArgList args, ConnectionContext* cntx) {
         // The response for 'capa dragonfly' is: <masterid> <syncid> <numthreads> <version>
         auto* rb = static_cast<RedisReplyBuilder*>(cntx->reply_builder());
         rb->StartArray(4);
-        rb->SendSimpleString(master_id_);
+        rb->SendSimpleString(master_replid_);
```

Comment:
I dont understand how this is the same as cluster node id ..
master_replid_ is set in this line
master_replid_ = GetRandomHex(eng, CONFIG_RUN_ID_SIZE);
I dont see where you that you overide it with cluster node id

Reply:
This PR splits the IDs into 2 separate ones, if `--cluster_node_id` is set.
I.e. the ID used for `master_replid_` is *always* random and can't be overridden, while the ID for cluster has a default value of `master_replid_` but *can* be overridden if set as an argument.

Reply:
Ok sorry I miss understood your response on the other thread. So if this is not the same id it as the master_replid_ we should say that we do not support this flag for emulated cluster because in emulated cluster we reply the replica / master  etl.remote_client_id_ which is the master_replid_ generated